torch/csrc/utils/python_tuples.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <Python.h>
+#include <sstream>
+#include "torch/csrc/utils/object_ptr.h"
+#include "torch/csrc/utils/python_numbers.h"
+
+inline void THPUtils_packInt64Array(PyObject *tuple, size_t size, const int64_t *sizes) {
+  for (size_t i = 0; i != size; ++i) {
+    PyObject *i64 = THPUtils_packInt64(sizes[i]);
+    if (!i64) {
```

Comment:
Just throw a `python_error`. Otherwise you clobber the underlying error (such as OutOfMemory) with this message.